src/StoreApi/Schemas/CartSchema.php

```diff
@@ -302,6 +302,12 @@ public function get_properties() {
 					'properties' => $this->force_schema_readonly( $this->error_schema->get_properties() ),
 				],
 			],
+			'payment_requirements'    => [
+				'description' => __( 'List of required payment gateway features to process the order.', 'woo-gutenberg-products-block' ),
+				'type'        => 'array',
+				'context'     => [ 'view', 'edit' ],
+				'readonly'    => true,
+			],
```

Comment:
Conceptually, this makes more sense to be within `CheckoutSchema` but due to limitations (we don't get an updated `checkout` response on each change in cart like shipping, items, coupons...), we have to leave it here.

Reply:
I agree. My first approach was with `CheckoutSchema` but hit a wall after I realized how things are set up :)

Reply:
Shouldn't this be an enum field?